src/compiler/checker.ts

```diff
@@ -34287,10 +34291,69 @@ namespace ts {
                     return isLiteralImportTypeNode(node) ? getSymbolAtLocation(node.argument.literal) : undefined;
 
                 case SyntaxKind.ExportKeyword:
-                    return isExportAssignment(node.parent) ? Debug.assertDefined(node.parent.symbol) : undefined;
+                    if (isExportAssignment(node.parent)) {
+                        return Debug.assertDefined(node.parent.symbol);
+                    }
+                    break;
+            }
+        }
 
-                default:
-                    return undefined;
+        /**

```

Comment:
I am not sure this needs to live here and change the Checker api.. Instead having it as utility function is better since it can always use getSymbolAtLocation?

Reply:
This also uses `getSymbolOfNode` which isn't exposed.